### PR TITLE
Gulp pipeline baby!

### PR DESCRIPTION
**Pull request by koddsson**
Check it out. It's pretty sweet IMHO.

cc/ @MiniGod @benediktvaldez @kristjanmik

### CODE REVIEW

**Comment by koddsson** [comment]
Oh baby. This is ready for review! :revolving_hearts:

**Comment by koddsson** [comment]
Maybe I should add documentation for this...

**Comment by koddsson** [comment]
BOOOOOM!

Somebody review me plz!

**Comment by koddsson** [comment]
I think I resolved both your questions @kristjanmik.

File: package.json [hunk 1]
```diff
@@ -1,25 +1,36 @@
 {
   "name": "v2apis",
-  "version": "0.0.0",
-  "description": "",
+  "version": "2.0.0",
+  "description": "The purpose of [APIs.is](http://apis.is) is to make data readily available masterrto anyone interested. All data that is delivered through APIs.is is JSON formatted and scraped from open public websites.",
```
Comment (kristjanmik) [inline package.json]: Does not support markdown

**Pull request merged**